go.mod

```diff
@@ -3,27 +3,27 @@ module github.com/openshift/client-go
 go 1.20
 
 require (
-	github.com/openshift/api v0.0.0-20230807121159-a81c3efc8824
+	github.com/openshift/api v0.0.0-20230911111751-da2f2ca9ae0f
 	github.com/openshift/build-machinery-go v0.0.0-20220913142420-e25cf57ea46d
 	github.com/spf13/pflag v1.0.5
-	k8s.io/api v0.27.2
-	k8s.io/apimachinery v0.27.2
-	k8s.io/client-go v0.27.1
-	k8s.io/code-generator v0.27.1
+	k8s.io/api v0.28.1
+	k8s.io/apimachinery v0.28.1
+	k8s.io/client-go v0.28.1
+	k8s.io/code-generator v0.28.1
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3
 )
 
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/emicklei/go-restful/v3 v3.9.0 // indirect
-	github.com/evanphx/json-patch v4.12.0+incompatible // indirect
+	github.com/evanphx/json-patch v5.6.0+incompatible // indirect
```

Comment:
We can't fix that easily, but it's a known regression that was fixed in https://github.com/kubernetes/kubernetes/pull/120327 we'll need to bump k8s deps to 1.28.2 later this week.